### DISCUSSION TITLE
interface DialogPortalProps not accepting className

### DISCUSSION THREAD

**Discussion by shryesh-khandelwal** [radix-ui/primitives#2449] [Help] [answered]
Encountered an issue with using shadcn/ui
The interface DialogPortalProps missing className in definition

Am unable to build my library without this error being fixed.

Any workarounds?

**Comment by joaom00** [comment, marked as answer]
See the fix in shadcn/ui repo https://github.com/shadcn-ui/ui/pull/1603
